cards without image uris.sql

```diff
@@ -0,0 +1,3 @@
+SELECT *
+FROM 'MTG-Cards'
+WHERE small_img IS NULL;
```

Comment:
Should we add .sql files to the gitignore?

Reply:
I added that file for the reviewer to validate the cards which have `card_faces` but do not have any image uris. 
The cards that have `card_faces` automatically don't come with any image uris so the `small_img` can only come from the `card_faces` values.